src/main.rs

```diff
@@ -153,16 +153,30 @@ enum Sevctl {
     },
 }
 
-fn download(
-    rsp: reqwest::Result<reqwest::blocking::Response>,
-    usage: Usage,
-) -> Result<sev::Certificate> {
-    let rsp = rsp.context(format!("unable to contact {} server", usage))?;
-
-    let status = rsp.status();
-    let mut rsp = rsp.error_for_status().context(&format!(
-        "received failure from {} server: {}",
-        usage, status
+use std::time::Duration;
+
+fn download(url: &str, usage: Usage) -> Result<sev::Certificate> {
+    let mut rsp = reqwest::blocking::get(url);
+    for request_wait_seconds in &[0, 2, 4, 6, 9] {
+        std::thread::sleep(Duration::from_secs(*request_wait_seconds));
+        match &rsp {
+            // Connection is established, ensure that status code is not an error.
+            Ok(found) => {
```

Comment:
This comment is a little misleading since as I understand it, the `reqwest::blocking::get()` is a one-shot HTTP request, meaning that we don't technically establish & maintain a connection after it's been called.

It would be more appropriate to write something about how the HTTP request appears to have succeeded, and we're just checking for an OK status code.